core/sql/cli/CliExpExchange.cpp

```diff
@@ -3804,7 +3893,10 @@ InputOutputExpr::inputValues(atp_struct *atp,
           if (!source) {
 	    continue;
 	  }
-
+          if(isOdbc)
```

Comment:
I wonder whether it is correct that JDBC passses a string of the format _iso88591'ABCD' here. I would have expected that trafci should have already converted this to an unquoted string. You can try the following with your fix and see what you get:

trafci
execute xx using '_iso88591''ABCD''';

Will this insert ABCD into the table or _iso88591'ABCD'? If it inserts ABCD, then I think the real bug is in trafci, not in this method, InputOutputExpr::inputValues().

Reply:
I have test if we not remove the prefix the prefix will insert into the table.
![tim 20180628093450](https://user-images.githubusercontent.com/4456500/42008145-905dec1e-7ab6-11e8-85e0-33109d345c2f.png)

Reply:
Maybe trafci does not require quotes around string literals in the "using" clause. Also, I don't think we should allow this kind of syntax in the "using" clause. We are specifying the value of a parameter that already has a data type. Specifying a character set at this time does not make sense, that syntax is meant for SQL queries, not for variables of parameters.

Note that the "using" clause does not accept SQL syntax, only values. So, execute xx using 3+4 would probably insert '3+4' into the database, not '7'.